CODE_OF_CONDUCT.md

```diff
@@ -17,9 +17,7 @@ The xi-editor project follows the Rust Code of Conduct, reproduced below.
 
 ## Moderation
 
-
-These are the policies for upholding our community's standards of conduct. If you feel that a thread needs moderation, please contact
-a mod (on IRC) or a maintainer (on Github). To contact a moderator by email, use [mods@xi-editor.io](mailto:mods@xi-editor.io).
+These are the policies for upholding our community's standards of conduct. If you feel that a thread needs moderation, please contact a maintainer on Github. To contact a moderator by email, use [mods@xi-editor.io](mailto:mods@xi-editor.io).
```

Comment:
github doesn't really have 'private messaging' so I'm not sure this makes sense. Maybe just list the email?

Reply:
Yes, that's probably more clear. I was thinking you could look up an email through Github, but that's alot of indirection. I'll fix this.